WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -1030,6 +1032,7 @@ - (void) uploadFeatureImage:(ALAsset *)asset
                           }];
         [progress setUserInfoObject:[UIImage imageWithData:media.thumbnail] forKey:WPProgressImageThumbnailKey];
         progress.localizedDescription = NSLocalizedString(@"Uploading...",@"Label to show while uploading media to server");
+        progress.kind = NSProgressKindFile;
```

Comment:
I'm not too familiar with the progress control so I'm also checking the docs to learn more. I noticed the docs say:
> If you set this value for the progress kind, you must set a value in the user info dictionary for the NSProgressFileOperationKindKey. 

I'm not sure, but should we add 
`[progress setUserInfoObject:NSProgressFileOperationKindCopying forKey:NSProgressFileOperationKindKey];`

It doesn't seem to make a difference if this is set in the user info dictionary or not as far as I can tell.